client/src/components/Toggle/index.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+class Toggle extends Component {
```

Comment:
@dropcat13 do we want to use a Component here? Or should we structure this like the rest of the app?